features/admin.ai.v1/providers/ai-branding-preference-provider.tsx

```diff
@@ -48,13 +50,17 @@ const AIBrandingPreferenceProvider: FunctionComponent<AIBrandingPreferenceProvid
 
     const { children } = props;
 
-    const { preference } = useBrandingPreference();
```

Comment:
Shall we update the PR description with the reason for opting out of retrieving branding preferences from the context?

Also, if we are not using the branding preferences from context anywhere, let's remove it from the context too.